src/backend/InvenTree/InvenTree/schema.py

```diff
@@ -101,6 +101,16 @@ def get_operation(
                         f'{parameter["description"]} Possible fields: {", ".join(ordering_fields)}.'
                     )
 
+        # Change return to array type, simply annotating this return type attempts to paginate, which doesn't work for
+        # a create method and removing the pagination also affects the list method
+        if self.method == 'POST' and type(self.view).__name__ == 'StockList':
```

Comment:
This seems very finicky; can this be addressed in/on the view itself?

Reply:
tbh I would prefer that approach in the future, this is a very untransparent way of handling this and very hard to discover